lib/still/data.ex

```diff
@@ -0,0 +1,162 @@
+defmodule Still.Data do
+  import Still.Utils
+
+  use GenServer
+
+  @supported_extensions [".yml", ".exs", ".ex", ".json"]
+
+  @initial_state %{global: %{}}
+
+  @default_folder "_data"
+
+  @moduledoc """
+  Loads data files and makes data available in templates.
+  Any file in the _#{@default_folder}_ folder will be loaded
+  using the file's name as a key. For instance, a file in
+  _#{@default_folder}/site.json_ with the contents:
+
+      {
+        "title": "Still"
+      }
+
+  Will be available in the templates as `@site.title`.
+
+  You can also use folders to organise files; the same file in
+  `_#{@default_folder}/default/site.json_ would be available
+  in the templates as `@default.site.title`.
```

Comment:
Just noticed this has formatting issues with the backticks

Reply:
👍